components/org.wso2.carbon.identity.oauth.ui/src/main/resources/web/oauth/add.jsp

```diff
@@ -357,37 +396,28 @@
                         document.getElementById("audience").value = "";
                         return false;
                     }
-
                     if ($.inArray(audience, audienceArr) !== -1) {
                         CARBON.showWarningDialog('<fmt:message key="duplicate.audience.value"/>');
                         document.getElementById("audience").value = "";
                         return false;
                     }
                     audienceArr.push(audience);
                     var propertyCount = document.getElementById("audiencePropertyCounter");
-
                     var i = propertyCount.value;
                     var currentCount = parseInt(i);
-
                     currentCount = currentCount + 1;
                     propertyCount.value = currentCount;
-
                     document.getElementById('audienceTableId').style.display = '';
                     var audienceTableTBody = document.getElementById('audienceTableTbody');
-
                     var audienceRow = document.createElement('tr');
                     audienceRow.setAttribute('id', 'audienceRow' + i);
-
                     var audiencePropertyTD = document.createElement('td');
                     audiencePropertyTD.setAttribute('style', 'color: rgb(119, 119, 119); font-style: italic;');
                     audiencePropertyTD.innerHTML = "" + audience + "<input type='hidden' name='audiencePropertyName' id='audiencePropertyName" + i + "'  value='" + audience + "'/> ";
-
                     var audienceRemoveTD = document.createElement('td');
                     audienceRemoveTD.innerHTML = "<a href='#' class='icon-link' style='background-image: url(../admin/images/delete.gif)' onclick='removeAudience(" + i + ");return false;'>" + "Delete" + "</a>";
-
                     audienceRow.appendChild(audiencePropertyTD);
                     audienceRow.appendChild(audienceRemoveTD);
-
```

Comment:
any reason to remove the new lines here?

Reply:
fixed with 95271dc3c7cc820ee8b77a92ce75c0129db0015b